### PR TITLE
117 tf 0 general

### PR DESCRIPTION
**Pull request by ToddCooper**
Non-Volume supplement content + Vol 0 content now complete.  

This can be pushed to the master OR the "yellow" highlights reviewed and discussed.

### CODE REVIEW

**Comment by ToddCooper** [comment]
Note:  Updated a couple of the files today AFTER the PR was generated.  See the latest update [ef3892c](https://github.com/IHE/sdpi-fhir/pull/127/commits/ef3892c1efbad2bf01b792564a22bde9b3425e90).

File: SDPi_Supplement/asciidoc/volume0/tf0-ch-9-copyrights.adoc [hunk 1]
```diff
@@ -8,5 +8,60 @@ NOTE:  Migrate content from the Word document here
...
+Some IHE Profiles incorporate SNOMED^®^ CT, which is used by permission of the International Health Terminology Standards Development Organisation. SNOMED CT© was originally created by the College of American Pathologists. SNOMED CT is a registered trademark of the International Health Terminology Standards Development Organisation, all rights reserved.
+
+[%noheader]
+//[%autowidth]
+[cols="1"]
+|===
+|Amend section 9.1.x by adding the following:
+|===
+
+==== IEEE 11073 (Health Device Interoperability)
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume0/tf0-ch-9-copyrights.adoc:40]: In this section some weird formatting is going on - numbers, indentations, bullet points. This requires some asciidoc tweaking

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc [hunk 2]
```diff
@@ -21,11 +21,14 @@ NOTE: the following section was moved from the mothership document to here for i
 [bibliography]
 === Referenced Standards
 
-* [[[ieee_11073_10207_2017,IEEE 11073-10207:2017]]] IEEE 11073-10207-2017, Health informatics — Point-of-care medical device communication — Part 10207: Domain Information and Service Model for Service-Oriented Point-of-Care Medical Device Communication, 2018-02, available at https://standards.ieee.org/standard/11073-10207-2017.html.
+##ADD REFERENCES FROM GLOSSARY
+##
 
-* [[[ws_discovery_2009, WS-Discovery:2009]]] Web Services Dynamic Discovery (WS-Discovery) Version 1.1, OASIS Standard, 1 July 2009, available at http://docs.oasis-open.org/ws-dd/discovery/1.1/wsdd-discovery-1.1-spec.html
+* [[[ieee_11073_10207_2017,IEEE 11073-10207:2017]]] IEEE 11073-10207-2017, Health informatics — Point-of-care medical device communication — Part 10207: Domain Information and Service Model for Service-Oriented Point-of-Care Medical Device Communication, 2018-02, available at https://standards.ieee.org/standard/11073-10207-2017.html.  footnote:ieee_permission[]
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc:27]: Is there a reason for the re-ordering? Does it follow a specific rule?

File: SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc [hunk 3]
```diff
@@ -13,6 +13,9 @@
 
 [sdpi_transaction#transaction_dev_23,actors='actor_somds_provider actor_somds_consumer',figure=vol2_figure_dev_23_sequence,messages='message_announce_network_presence']
 
+##NOTE THAT THE FIRST SENTENCE SHOULD BE THE SUMMARY THAT IS PUBLISHED IN THE TF-0 APPENDIX B TRANSACTIONS TABLE - but substituting with first sentence and not reftext ... HOW???
+##
+[[transaction_summary_dev_23,DEV-23 Announce Network Presence]]
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc:18]: What kind of markup is that?

File: SDPi_Supplement/asciidoc/volume0/tf0-ch-9-copyrights.adoc (hunk 1 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume0/tf0-ch-9-copyrights.adoc:40]: Not sure of the "weird" part but the specific section you identified was the 1-cell table that included the Editor Instructions from the Word template.

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc (hunk 2 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc:27]: 1st - IHE profiles / TF's have no place for including references such as this, other than the various standards conformance sections associated with profiles and transactions.  The only change should have been adding the footnote: macro.  Was the actual sequencing of 26 and 27 changed?  If so ... it wasn't intentional!!!

File: SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc (hunk 3 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc:18]: This ## ... ##  was inserted by IntelliJ when I wanted to highlight (yellow) a section of a block.  Note that in branch 117, esp. Supplement & Volume 0 stuff, I integrated highlighted comments for items that either I wanted to make sure we discussed, or that I didn't have a clue HOW to do something (more likely the case!)  

IF the question is around the insertion of the transaction_summary_ block anchor, we can talk about the best way of formalizing AND where to formalize the summary language, which is in the TF-0 Appendix B file

**Review by d-gregorczyk — APPROVED** [review]

**Pull request merged**